packages/contracts-bedrock/semver-lock.json

```diff
@@ -33,7 +33,7 @@
   },
   "src/L1/OPStackManager.sol": {
     "initCodeHash": "0x31378d0ea88a358f517927f5bc530a5b958f43c5462e4a822388ec70bd45e8d4",
-    "sourceCodeHash": "0xf8ac1ce2fbaa9aa21bd6c60e0bf863b06ea3bdd1c4ed801ed70ee64c1b534f67"
+    "sourceCodeHash": "0xa4826deb93a7fe55ebd3bc16610ce6d42e5c85d88ce960ec5dcd941c0018c124"
```

Comment:
I think the OPSM needs a beta version bump based on [my reading](https://github.com/ethereum-optimism/design-docs/blob/main/protocol/smart-contract-versioning.md#individual-contract-versioning).

Reply:
Good call. Though after https://github.com/ethereum-optimism/optimism/pull/11943#discussion_r1763768158 there is no longer a diff to OPSM or semver-lock, so this is no longer necessary